doc/ug/_templates/custom_links_sidebar.html

```diff
@@ -1 +1,2 @@
+<a href="doxygen">Doxygen</a>
 <a href="http://www.espresso-pp.de/Impressum.html">Impressum</a>
```

Comment:
If we have more links in this part then let's change it into list
```html
<ul>
<li><a href="doxygen/index.html">Doxygen</a></li>
<li><a href="http://www.espresso-pp.de/Impressum.html">Impressum</a></li>
</ul>
```

btw, @stuehn what is this Impressum.html ? on main page it's redirect to ```http://www.mpip-mainz.mpg.de/2331/imprint```